mq/publishers.go

```diff
@@ -7,18 +7,23 @@ import (
 	"sync"
 	"time"
 
+	"github.com/google/uuid"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/gravitl/netmaker/logic"
 	"github.com/gravitl/netmaker/models"
 	"github.com/gravitl/netmaker/servercfg"
 	"golang.org/x/exp/slog"
 )
 
-var batchSize = servercfg.GetPeerUpdateBatchSize()
-var batchUpdate = servercfg.GetBatchPeerUpdate()
+var running bool
 
 // PublishPeerUpdate --- determines and publishes a peer update to all the hosts
 func PublishPeerUpdate(replacePeers bool) error {
+	if running {
+		return nil
+	}
+	running = true
+
```

Comment:
can we remove this check for now?
we need to add a better logic to throttle peer updates, can be handled in next cycle

Reply:
Yes, removed in new commit as well the code conflict, please review again.